.gitignore

```diff
@@ -1,27 +1,106 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# I don't konw.
 _trial_temp
-build
-build-pypy
-MANIFEST
-dist
 apidocs
 *,cover
 .testrepository
-*.pyc
-*.pyd
-*.pyo
-*.so
 *~
-*.swp
-*.swh
-*.swn
-*.swo
+*.sw[phno]
 docs/tutorial/index.html
 dulwich.egg-info/
+docs/api/*.txt
```

Comment:
These files are checked in and should not be ignored.

Reply:
![Screenshot_20201201-154236](https://user-images.githubusercontent.com/38396747/100712026-0fc88f80-33ed-11eb-9c16-c5a6c14ad30d.png)

This is your own original ignore list. I just moved the position. Many of them are originally there, but you said you want to delete them.